pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -261,102 +293,108 @@ public void start() throws PulsarServerException {
+                    state -> {
+                        pulsar.getLoadManagerExecutor().execute(() -> {
+                            if (state == LeaderElectionState.Leading) {
+                                playLeader();
+                            } else {
+                                playFollower();
+                            }
+                        });
                     });
-                });
-        this.serviceUnitStateChannel = new ServiceUnitStateChannelImpl(pulsar);
-        this.brokerRegistry.start();
-        this.splitManager = new SplitManager(splitCounter);
-        this.unloadManager = new UnloadManager(unloadCounter);
-        this.serviceUnitStateChannel.listen(unloadManager);
-        this.serviceUnitStateChannel.listen(splitManager);
-        this.leaderElectionService.start();
-        this.serviceUnitStateChannel.start();
-        this.antiAffinityGroupPolicyHelper =
-                new AntiAffinityGroupPolicyHelper(pulsar, serviceUnitStateChannel);
-        antiAffinityGroupPolicyHelper.listenFailureDomainUpdate();
-        this.antiAffinityGroupPolicyFilter = new AntiAffinityGroupPolicyFilter(antiAffinityGroupPolicyHelper);
-        this.brokerFilterPipeline.add(antiAffinityGroupPolicyFilter);
-        SimpleResourceAllocationPolicies policies = new SimpleResourceAllocationPolicies(pulsar);
-        this.isolationPoliciesHelper = new IsolationPoliciesHelper(policies);
-        this.brokerFilterPipeline.add(new BrokerIsolationPoliciesFilter(isolationPoliciesHelper));
-
-        createSystemTopic(pulsar, BROKER_LOAD_DATA_STORE_TOPIC);
-        createSystemTopic(pulsar, TOP_BUNDLES_LOAD_DATA_STORE_TOPIC);
+            this.serviceUnitStateChannel = new ServiceUnitStateChannelImpl(pulsar);
+            this.brokerRegistry.start();
+            this.splitManager = new SplitManager(splitCounter);
+            this.unloadManager = new UnloadManager(unloadCounter);
+            this.serviceUnitStateChannel.listen(unloadManager);
+            this.serviceUnitStateChannel.listen(splitManager);
+            this.leaderElectionService.start();
+            this.serviceUnitStateChannel.start();
+            this.antiAffinityGroupPolicyHelper =
+                    new AntiAffinityGroupPolicyHelper(pulsar, serviceUnitStateChannel);
+            antiAffinityGroupPolicyHelper.listenFailureDomainUpdate();
+            this.antiAffinityGroupPolicyFilter = new AntiAffinityGroupPolicyFilter(antiAffinityGroupPolicyHelper);
+            this.brokerFilterPipeline.add(antiAffinityGroupPolicyFilter);
+            SimpleResourceAllocationPolicies policies = new SimpleResourceAllocationPolicies(pulsar);
+            this.isolationPoliciesHelper = new IsolationPoliciesHelper(policies);
+            this.brokerFilterPipeline.add(new BrokerIsolationPoliciesFilter(isolationPoliciesHelper));
+
+            createSystemTopic(pulsar, BROKER_LOAD_DATA_STORE_TOPIC);
```

Comment:
@heesung-sn Somehow, the creation topic might time out, so the broker will shut down, but when `brokerRegistry` does not close, the zk path exists, and the broker start will fail again.

Reply:
Do you know why the topic creation will time out? I saw several times, but after broker restarted, it will be successful start.